.circleci/config.yml

```diff
@@ -31,17 +31,17 @@ jobs:
             - "/usr/local/bin"
             - "/usr/local/lib/python3.7/site-packages"
 
-      - run: pipenv run bash ./bin/lint
-      - run: pipenv run bash ./bin/coverage
+      - run: bash ./bin/lint
+      - run: bash ./bin/coverage
       - store_artifacts:
           path: ./htmlcov
           destination: coverage
       - run:
           name: Initialize Elastic Beanstalk
-          command: pipenv run eb init -p docker project-confero
+          command: eb init -p docker project-confero
       - deploy:
           name: Deploy to Elastic Beanstalk Environment
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
-              pipenv run eb deploy --region us-east-2
+              eb deploy --region us-east-2
             fi
```

Comment:
Just a thought, but shouldn't we have the `./manage.py collectstatic` command somewhere in here so stuff doesn't get left out by mistake?

Reply:
aaaah, I see it in the Dockerfile now. Could that create weird issues since we're copying in the code, not mounting it as a volume? If it works with `./bin/docker-tests` it'll probably be fine, but I haven't seen it done like that.

Reply:
Yeah I don't love it either. But I spent some time trying to get this running in an `.ebextension` `container_command`, without success. Then I realized that all I was trying to do was to run this code in the docker container, so why not do it directly?

And we could do it in Circle, but looks like by default ` eb deploy` just deploys a zip of all your git committed files. There's a way to override that with a `.ebignore` file, but I could imagine that going wrong when we update the `gitignore` and forget to update the `.ebignore`. 

Also, I just noticed that `.ebextensions/db-migrate.config` seems to already be configured to run `collectstatic`, but it's apparently not working, since without this last line the static files don't get served.

Anyway tl;dr - maybe the next time we're messing with the EB config we can figure out a better way to do this